ci/tests/backend/alba.py

```diff
@@ -240,8 +245,31 @@ def is_bucket_count_valid_with_policy(bucket_count, policies):
     return safe
 
 
+def initialise_disks(alba_node):
+    disks_to_init = [d['name'] for d in alba_node.all_disks if d['available'] is True]
+    failures = AlbaNodeController.initialize_disks(alba_node.guid, disks_to_init)
+    if failures:
+        raise 'Alba disk initialization failed for (some) disks: {0}'.format(failures)
```

Comment:
To be slightly more clear, this raise will fail and will in turn raise a new exception:

```
TypeError: exceptions must be old-style classes or derived from BaseException, not str
```